src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -513,10 +525,6 @@ private static Collection<ScriptBindingProvider> LoadBindingProviders(ScriptHost
             // Register our built in extensions
             var bindingProviderTypes = new Collection<Type>()
             {
-                // binding providers defined in this assembly
```

Comment:
eventually, we should be able to remove this entire list and everything is self-describing.